admin/templates/register.html

```diff
@@ -0,0 +1,73 @@
+{% load render_bundle from webpack_loader %}
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>OSFAdmin | Log in</title>
+    <!-- Tell the browser to be responsive to screen width -->
+    <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
+    <!-- Bootstrap 3.3.4 -->
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet" type="text/css" />
+    <!-- Font Awesome Icons -->
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
+    <!-- Theme style -->
+    <link href="{% static 'css/AdminLTE.min.css' %}" rel="stylesheet" type="text/css" />
+
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+ <body class="register-page">
+  <div class="register-box">
+    <div class="register-logo">
+      <b>OSF</b>Admin
+    </div>
+    <div class="register-box-body">
+      <p class="login-box-msg">Register a new account</p>
```

Comment:
In order to see if there were any non-field-specific errors I added this:
```html
      <div>
          {{ form.errors }}
      </div>
```

You could make it much prettier and make it only show non-field errors. This just got the job done for debugging. Here's some documentation on it. https://docs.djangoproject.com/en/1.9/topics/forms/#rendering-form-error-messages